topo/node/gobgp/gobgp.go

```diff
@@ -27,8 +27,6 @@ func New(nodeImpl *node.Impl) (node.Node, error) {
 	if nodeImpl.Proto == nil {
 		return nil, fmt.Errorf("nodeImpl.Proto cannot be nil")
 	}
-	cfg := defaults(nodeImpl.Proto)
```

Comment:
We do not need any defaults in this node ?

Reply:
You are right, we would need those defaults. I will update.